src/converters/lintConfigs/rules/convertRules.ts

```diff
@@ -78,21 +80,26 @@ export const convertRules = (
                     continue;
                 }
 
-                // 4d. If not, a rule merger is run to combine it with its existing output settings.
+                // 4d. First it merges the notices, since that is always needed.
```

Comment:
These intentionally reflect what's in `docs/Architecture/Linters.md`. Could you please update that file too?

Also, nit, phrases like "is always needed" are generally implied by the context.

```suggestion
                // 4d. First it deduplicates and merges notices
```